src/components/Drawer.js

```diff
@@ -131,31 +126,31 @@ const Drawer = React.createClass({
           type='base'
         />
       </nav>
-    ) : <div style={styles.nav} />;
+    ) : <div className={css(styles.nav)} />;
   },
 
   render () {
     const styles = this.styles();
```

Comment:
Actually it looks like in this case that wouldn't be an issue, and it may be possible to use inline styles in combination with Aphrodite style sheets for the dynamic values we do need?